hangman.js

```diff
@@ -1 +1,72 @@
-console.log("welcome to hangman");
\ No newline at end of file
+//Test area
+console.log("welcome to hangman");
+alert("Hello!");
+
+
+//Do I want to create an object?
+
+
+
+var secretWord;
+var playerGuess;
+var score = [];
```

Comment:
Could probably pick a better name for this. `score` usually indicates some sort of numerical value.